src/gui/search/searchjobwidget.cpp

```diff
@@ -390,12 +390,12 @@ void SearchJobWidget::contextMenuEvent(QContextMenuEvent *event)
     auto *menu = new QMenu(this);
     menu->setAttribute(Qt::WA_DeleteOnClose);
 
-    menu->addAction(UIThemeManager::instance()->getIcon(u"kt-set-max-download-speed"_qs)
+    menu->addAction(UIThemeManager::instance()->getIcon(u"download"_qs)
```

Comment:
I would rename to `speed-full` (and `speed-limited` for the other part) or anything like that. The current `download` is too generic and it doesn't relate to speed nor speedometer.

Reply:
`speed-full` then, what about `kt-set-max-upload-speed`?

Reply:
Sorry, disregard my previous post.